daprdocs/content/en/python-sdk-docs/python-client.md

```diff
@@ -66,6 +69,9 @@ with DaprClient() as d:
 The legacy environment variables `DAPR_RUNTIME_HOST`, `DAPR_HTTP_PORT` and `DAPR_GRPC_PORT` are 
 also supported, but `DAPR_GRPC_ENDPOINT` takes precedence.
 
+##### Dapr API Token
+
+You can use the `DAPR_API_TOKEN` environment variable to set the Dapr API token. 
```

Comment:
@elena-kolevska what do you think about linking to [Enable API token authentication in Dapr](https://docs.dapr.io/operations/security/api-token/), or the [Environment variables](https://docs.dapr.io/reference/environment/) doc?

Reply:
Great idea, thanks! I added it.